python/ray/ml/predictors/integrations/tensorflow/tensorflow_predictor.py

```diff
@@ -121,4 +132,20 @@ def build_model(self):
         Returns:
             DataBatchType: Prediction result.
         """
-        raise NotImplementedError
+        data = self.preprocessor.transform_batch(data)
+
+        if isinstance(data, pd.DataFrame):
+            if feature_columns:
+                data = data[feature_columns]
+            data = data.values
+        else:
+            data = data[:, feature_columns]
+
+        tensor = tf.convert_to_tensor(data, dtype=dtype)
+
+        model = self.model_definition()
```

Comment:
Can you leave a comment saying that the reason we are not doing this in __init__ is because this would make the predictor unserializable?